exercises/07-normalizr/backend/server.ts

```diff
@@ -0,0 +1,77 @@
+import express from 'express'
+
+import { User, UserData } from '../src/modules/users/user-types'
+
+const port = 3001
+
+const areStringsSame = (a: string, b: string): boolean =>
+  a.toLowerCase() === b.toLowerCase()
+
+const hasSameName = (
```

Comment:
I'd consider `hasSameName(user1: UserData, user2: UserData)`

Reply:
1. I propose to rename `UserData` interface to `UserName`, because it contains only `firstName` and `lastName`. This will affect multiple exercises. I'd use this interface for both arguments.
2. Is `hasSameName` name OK? Shouldn't it also reflect case insensitivity? E.g. `hasSameNameCaseInsensitive`

Reply:
1. agreed.
2. i'm fine with both. I'd prefer hasSameName in this context (I don't know why, I just would 🤷 )

Reply:
Rewritten with both arguments of type `UserData`, see https://github.com/salsita/react-training/pull/59/commits/c7b960b599747f88aebd2d95c3e13abc81385778

Renamed the `UserData` interface to `UserName` in 897bd26